src/main/java/org/alfresco/util/ConfigScheduler.java

```diff
@@ -198,24 +210,28 @@ private void clearPreviousSchedule()
      */
     public boolean readConfigAndReplace(boolean scheduledRead)
     {
-        boolean successReadingConfig;
-        log.debug((scheduledRead ? "Scheduled" : "Unscheduled")+" config read started");
-        Data data = getData();
-        try
-        {
-            Data newData = createData();
-            threadData.set(newData);
-            successReadingConfig = readConfig();
-            data = newData;
-            log.debug("Config read finished "+data+
-                    (successReadingConfig ? "" : ". Config replaced but there were problems")+"\n");
-        }
-        catch (Exception e)
+        boolean successReadingConfig = true;
```

Comment:
As successReadingConfig is now initialised to true should it be renamed as it will be true even if there was no attempt to read the config?

Reply:
Added a comment rather than changing the name.